.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_VERSION=2.0.0
```

Comment:
Please create .env.local file instead of .env, and make sure to not push it on github.